Source/WebKit/NetworkProcess/curl/NetworkDataTaskCurl.cpp

```diff
@@ -212,7 +212,7 @@ void NetworkDataTaskCurl::curlDidReceiveData(CurlRequest&, Ref<SharedBuffer>&& b
     }
 
     // FIXME: This should be exact network transferred size rather than decoded bytes.
-    setTotalBytesTransferredOverNetwork(totalBytesTransferredOverNetwork() + buffer->size());
+    setBytesTransferredOverNetwork(0);
```

Comment:
Seems like it would make more sense to just remove these lines entirely from the curl and soup impls. The ports can add support for this feature if they want on their own.